Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -8,6 +8,9 @@
   - type: SolutionContainerManager
   - type: Sprite
     state: produce
+  # let cows eat raw produce like wheat and oats
+  - type: Food
+    stomachs: 2
```

Comment:
Isn't this field `requiredStomachs`? Was humans eating wheat tested?

Reply:
havent tested people eating it but its using same field name as kudzu

Reply:
You should really test things. This is how we get lots of fun merged!

https://github.com/space-wizards/space-station-14/assets/3229565/3182996f-99be-4ef2-b8db-63085761d694

Reply:
P.S. I saw what the original comment was :D

Reply:
yes i thought

anyway it might just be worth letting anything eat produce since chickens dont have 2 stomachs and chewing wheat as le rp thing

Reply:
> yes i thought
> 
> anyway it might just be worth letting anything eat produce since chickens dont have 2 stomachs and chewing wheat as le rp thing

What no, how is this RP? People can't eat wheat. That's why we have chefs.

Reply:
chewing wheat farmer stereotype thing not speedrun eating it but yeah

Reply:
Please don't explain away a regression by saying this was intended. Wheat wasn't made edible in the before times for a reason. Unless there is a very good reason to change that, let's not introduce regressions.

Reply:
not saying intended just funny way of looking at it anyway it fixed